### PR TITLE
Adding chemicalfun reaction generator

### PR DESCRIPTION
**Pull request by gdmiron**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dimitrikulik** [comment]
I agree to be listed as maintainer of the conda recipe for chemicalfun.

**Comment by sdmytrievs** [comment]
I agree to be listed in the maintainer's list.

File: recipes/chemicalfun/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+# Configure step
+cmake -DPYTHON_EXECUTABLE:FILEPATH="$PYTHON" \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+      -DCMAKE_INSTALL_LIBDIR=$PREFIX/lib \
+      ..
+# Build step
+make -j${CPU_COUNT}
+make install
```
Comment (ocefpaf) [inline recipes/chemicalfun/build.sh:20]: Is there a `make check` or `ctest` that we can run here?

Comment (gdmiron) [reply recipes/chemicalfun/build.sh:20]: @ocefpaf Thanks for the review. The only tests I added are in the meta.yaml, checking that the files are installed in the right place and that the python module imports correctly. If necessary, I can try to add additional tests using ctest, or from python pytest.

Comment (ocefpaf) [reply recipes/chemicalfun/build.sh:20]: If those are not available from upstream we don't really require those here. I'll merge it and defer to your judgment in the feedstock.

**Pull request merged**